Tests/Unit/EmogrifierTest.php

```diff
@@ -750,6 +756,43 @@ public function emogrifyFormatsCssDeclarations($cssDeclarationBlock, $expectedSt
     }
 
     /**
+     * Data provider for emogrifyInvalidDeclaration.
+     *
+     * @return array[]
```

Comment:
"an array of (associative) arrays whose values are string" is what string[][] means (mind the two pairs of square brackets). :-)

(We'll also try to get rid of all param/return array and make them more specific: #195)